bundles/org.openhab.binding.solarman/src/main/java/org/openhab/binding/solarman/internal/SolarmanLoggerHandler.java

```diff
@@ -94,7 +95,12 @@ public void initialize() {
                 logger.debug("Found definition for {}", config.inverterType);
             }
         }
-        SolarmanV5Protocol solarmanV5Protocol = new SolarmanV5Protocol(config);
+
+        if (logger.isDebugEnabled()) {
+            logger.debug("Raw Type {}", config.solarmanLoggerMode);
+        }
```

Comment:
Checking the logger mode is only needed for relative expensive operations. this `enum.toString` is not expensive.
```suggestion
        logger.debug("Raw Type {}", config.solarmanLoggerMode);
```

Reply:
Yes you are rigth, this ist not expensive so I agree with your suggested chage